src/isar/config/settings.env

```diff
@@ -1,4 +1,4 @@
-ISAR_ROBOT_PACKAGE = isar_robot
+ISAR_ROBOT_PACKAGE = isar_taurob
```

Comment:
```suggestion
ISAR_ROBOT_PACKAGE = isar_robot
```

Reply:
Default should be isar_robot

Reply:
Pressure is not included in this PR. 

"And future robot-dependent checks now all have to be defined as exceptions in ISAR, instead of a scalable function that can be defined in each isar-robot repo." - I reckon that's how it is handled in this PR. 

"These 3 commits are essentially one, I think they should be squashed as they solve one thing, not 3." - I Agree.